### PR TITLE
Fix mixed content and update outdated URLs

### PR DESCRIPTION
**Pull request by tonyseek**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -83,11 +83,11 @@ <h1 id="logo" class="span4"><a href="/"><img width="300" height="62" src="{{ STA
...
                     &copy; 2018 www.beijing-open-party.com
                     <br/>
-                    <a href="http://www.github.com/openparty/openparty" target="_blank">OpenParty App</a>
+                    <a href="https://github.com/openparty/openparty" target="_blank">OpenParty App</a>
                     Powered by
-                    <a href="http://djangoproject.com" target="_blank">Django</a>,
-                    <a href="http://twitter.github.com/bootstrap/" target="_blank">Bootstrap</a>,
-                    <a href="http://typo.sofish.de/" target="_blank">Typo</a>
+                    <a href="https://www.djangoproject.com" target="_blank">Django</a>,
+                    <a href="https://twitter.github.com/bootstrap/" target="_blank">Bootstrap</a>,
```
Comment (CNBorn) [inline templates/base.html]: 其它都改了的话这个也麻烦改成 https://getbootstrap.com/ 吧

File: templates/core/about.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <h2>OpenParty简介</h2> 
 <p>Open Party是由北京的多个技术社区共同举办的社交活动。活动本着自由、开放、分享、互动的宗旨，通过采用“非会议（Unconference）”的形式， 为参与人员创建一个表达和交流的平台。本活动的目标是为北京的IT从业人员营造一个交流、表现、学习、生活、发展、进步的空间。活动话题不仅仅局限于技术，还包括生活、旅游、艺术、创业、商务、投资、营销等各方面的话题。</p> 
 <h2>OpenParty历史</h2> 
-<p>详见：<a id="tcwz" title="http://www.beijing-open-party.com/index.php/category/beijingopenparty" href="../index.php/category/beijingopenparty" target="_blank">http://www.beijing-open-party.com/index.php/category/beijingopenparty</a></p> 
+<p>详见：<a id="tcwz" title="https://www.beijing-open-party.com/index.php/category/beijingopenparty" href="../index.php/category/beijingopenparty" target="_blank">https://www.beijing-open-party.com/index.php/category/beijingopenparty</a></p>
```
Comment (CNBorn) [inline templates/core/about.html]: not related to this PR: cc @Tin invalid URL

File: templates/base.html (hunk 1 above)
Comment (tonyseek) [reply templates/base.html]: 好

Comment (tonyseek) [reply templates/base.html]: 7671f08

**Pull request merged**